README.md

```diff
@@ -42,6 +42,16 @@ The `vrt.sh` script is meant to be called regularly via a Cronjob (every 5 hours
 or more, as the datasets will be downloaded and published again at each run),
 with the previous variables defined.
 
+## Optional variables
+
+The following environment variables are optional:
+
+```
+# export ACTIVATE_GEOMETRY_TYPE=false 
+True by default
+```
+
```

Comment:
More doc is needed. We should at least explain what it does. From the gdal doc: "PROMOTE_TO_MULTI can be used to automatically promote layers that mix polygon or multipolygons to multipolygons, and layers that mix linestrings or multilinestrings to multilinestrings. Can be useful when converting shapefiles to PostGIS and other target drivers that implement strict checks for geometry types"